kvbc/test/categorization/block_merkle_category_unit_test.cpp

```diff
@@ -97,12 +97,12 @@ TEST_F(block_merkle_category, put_and_get) {
   auto expected = MerkleValue{{block_id, val1}};
 
   // Get by key works
-  ASSERT_EQ(expected, cat.getLatest(key1).value());
+  ASSERT_EQ(expected, asMerkle(cat.getLatest(key1).value()));
   // Get by specific block works
-  ASSERT_EQ(expected, cat.get(key1, 1));
+  ASSERT_EQ(expected, asMerkle(cat.get(key1, 1)));
```

Comment:
I know it is not related to this particular PR, but calling `asMerkle(cat.get())` has a different effect to calling `asMerkle(cat.get().value())` if the optional is empty as the first one will be undefined behavior and the second one will throw an exception. We might miss some issues or have the test fail for strange reasons if we use the first format.

Reply:
I copy-pasted it from the other categories.
I removed the functions the accepts optional, but for the other categories, I guess it should be done as well.